pages/common/behat.md

```diff
@@ -0,0 +1,28 @@
+# behat
+
+> A PHP framework for automatically testing business expectations.
```

Comment:
Behavior-Driven Development framework for PHP

Reply:
```suggestion
> Behavior-Driven Development framework for PHP.
```

Reply:
Another: `A PHP framework for Behavior-Driven Development`